src/metaBlob.cxx

```diff
@@ -134,12 +134,18 @@ PointDim(void) const
 }
 
 void MetaBlob::
-NPoints(size_t npnt)
+NPoints(int npnt)
 {
   m_NPoints = npnt;
 }
 
-size_t MetaBlob::
+void MetaBlob::
+NPoints(size_t npnt)
```

Comment:
I think we decided to keep all as int.

s

On Thu, Mar 24, 2016 at 10:30 AM, Matt McCormick <notifications@github.com>
wrote:

> In src/metaBlob.cxx
> <https://github.com/Kitware/MetaIO/pull/19#discussion_r57324603>:
>
> >  {
> >    m_NPoints = npnt;
> >  }
> >
> > -size_t MetaBlob::
> > +void MetaBlob::
> > +NPoints(size_t npnt)
>
> This may not work on a 32-bit system build because size_t and int will
> have the same size. Please verify that the different sign is enough to
> prevent build errors on 32-bit builds.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Kitware/MetaIO/pull/19/files/161178a3b4fe69c8805e20cb958b9c1eaa0b7b2b#r57324603>
>



-- 
====================================================
Stephen R. Aylward, Ph.D.
Senior Director of Operations, North Carolina, Kitware, Inc.
http://www.kitware.com and http://www.aylward.org
(919) 969-6990 x300
====================================================